============================= test session starts =============================
platform win32 -- Python 3.12.0, pytest-7.4.3, pluggy-1.3.0
rootdir: C:\Users\jange\Python Scripts\arxiv-db
plugins: anyio-3.7.1, cov-4.1.0
collected 6 items

mongodb_api\tests\test_routes.py ......                                  [100%]

============================== warnings summary ===============================
..\..\anaconda3\envs\arxiv-db\Lib\site-packages\dateutil\tz\tz.py:37
  C:\Users\jange\anaconda3\envs\arxiv-db\Lib\site-packages\dateutil\tz\tz.py:37: DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.fromtimestamp(timestamp, datetime.UTC).
    EPOCH = datetime.datetime.utcfromtimestamp(0)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html

---------- coverage: platform win32, python 3.12.0-final-0 -----------
Name                               Stmts   Miss  Cover   Missing
----------------------------------------------------------------
mongodb_api\__init__.py                0      0   100%
mongodb_api\main.py                   30     11    63%   65-80
mongodb_api\models\__init__.py         0      0   100%
mongodb_api\models\models.py          33      1    97%   90
mongodb_api\routes.py                 84     30    64%   45-76, 94-96, 125-127, 175, 183, 214-216
mongodb_api\tests\__init__.py          0      0   100%
mongodb_api\tests\test_routes.py      48      0   100%
mongodb_api\utils.py                  24      1    96%   28
----------------------------------------------------------------
TOTAL                                219     43    80%

======================== 6 passed, 1 warning in 2.95s =========================
